### PR TITLE
Added support for scaledown using shrink playbooks in RHCS4 for all 

### PR DESCRIPTION
**Pull request by Ameenasuhani**
# Description
Fixes issue: https://github.com/red-hat-storage/cephci/issues/226

Signed-off-by: Ameenasuhani <ameenasuhani@gmail.com>

Logs: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-2OLCV4/

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarin Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by Ameenasuhani** [comment]
Added new polarion testcases:
https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83574769
https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83574770
https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83574771

File: tests/ceph_ansible/shrink_daemon.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        cmd += f" -e {daemon}_to_kill={daemons_to_kill} -i hosts"
+    elif daemon == "rgw":
+        cmd += f" -e {daemon}_to_kill={node_name}.rgw{instance} -i hosts"
+    else:
+        cmd += f" -e {daemon}_to_kill={node_name} -i hosts"
+
+    out, err = ceph_installer.exec_command(cmd=cmd, long_running=True)
+
+    if err != 0:
+        log.error("Failed during ansible playbook run\n")
```
Comment (manasagowri) [inline tests/ceph_ansible/shrink_daemon.py:59]: Could you please add the name of the playbook which failed.

**Review by manasagowri — CHANGES_REQUESTED** [review]
Overall looks good. I feel more info logs can be added in shrink_daemon.py for the user to know what is being done, which would help during debugging.

**Review by manasagowri — COMMENTED** [review]
Looks good to me.

**Review by manasagowri — DISMISSED** [review]

**Comment by sunilkumarn417** [comment]
@Ameenasuhani , can you please squash your commits?

**Comment by Ameenasuhani** [comment]
@sunilkumarn417 Done

**Review by psathyan — APPROVED** [review]

**Pull request merged**